### ISSUE TITLE
Have label-content-name-mismatch rule raise support issues

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#5228]
WCAG's requirement of the visible label needing to be somewhere in the accessible name doesn't match reality. The testing I did today suggests that on MacOS, Voice Control requires you to announce the full accessible name. Announcing only the first word, or the first few words does not work. 

I've read documentation that suggests iOS's Voice Control does allow it. I can't confirm that, but it's odd. I've in the past heard that Voice Assistant on Android, and Dragon on Windows will work on the visible text, so for them it doesn't matter if the text is in the accessible name or not.

Voice Control has number labels, which seem like a reasonable alternative for needing to announce the full accessible name. But how exactly that works with WCAG conformance is an open question to me. Arguably if something doesn't work in assistive tech it isn't accessibility support, and therefor fails WCAG.

## IMPORTANT

Research into this is ongoing. Not ready to be picked up.

**Comment by WilcoFiers** [comment]
Research done. We're not going to do this. Support for the current approach is sufficient. Apple's Voice Control is an outlier in that on Mac it requires the full name to be announced (and so the visible text must equal the name) and for iOS it requires announcing the start.

Since VoiceControl isn't on Axe-core's accessibility support list we're not going to raise that as an issue.
